lib/screens/explore_screens/explore/explore_screen.dart

```diff
@@ -97,7 +97,15 @@ class ExploreScreen extends StatelessWidget {
                       ),
                     ),
                     const SizedBox(width: 20),
-                    const Expanded(child: SizedBox.shrink()),
+                    Expanded(
```

Comment:
This needs to be behind a feature flag.

Reply:
Do I still need a flag if is going to be ready by sunday?

Reply:
no, if its going to be fully functional you are good without a feature flag

Reply:
well thats the goal (: